src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -0,0 +1,513 @@
+#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
```

Comment:
Can you fix the formatting of this file? It appears to be cut-and-pasted from the old ones.